src/styles/search.css

```diff
@@ -0,0 +1,35 @@
+.search-input {
+  width: 30%;
```

Comment:
Setting a static width for all screen sizes like this isnt the best idea, as it will look very small on mobile devices. A good approach for width is to set boundaries using properties like `min-width` and `max-width` as well as the standard `width` property set to 100%, something like:

```
min-width: 280px;
max-width: 800px;
width: 100%;
```
This means you stay within a given range and the element is responsive